appveyor.bat

```diff
@@ -330,7 +330,8 @@ copy /Y winpty* ..\..\
 set dir=vim%TAG_NAME:~1,1%%TAG_NAME:~3,1%
 mkdir ..\vim\%dir%
 xcopy ..\runtime ..\vim\%dir% /Y /E /V /I /H /R /Q
-7z a ..\..\gvim_%TAG_NAME:~1%_%ARCH%.zip ..\vim
+:: need to escape the ! because of delayedexpansion being active
+7z a ..\..\gvim_%TAG_NAME:~1%_%ARCH%.zip ..\vim -x^^!vim\%dir%\syntax\testdir\
```

Comment:
How about removing `testdir` after xcopy?
```batch
rd /S /Q ..\vim\%dir%\syntax\testdir
```

Reply:
yeah, that would have worked as well. Now I already merged it